Lib/test/test_sysconfig.py

```diff
@@ -346,12 +350,17 @@ def test_get_platform(self):
         # XXX more platforms to tests here
 
     @unittest.skipIf(is_wasi, "Incompatible with WASI mapdir and OOT builds")
+    @unittest.skipIf(is_apple_mobile,
+                     f"{sys.platform} doesn't distribute config.h in the runtime environment")
```

Comment:
The config folder isn't created as part of the iOS build. It's build by the `libainstall` target, which isn't part of an iOS build, since it includes binaries and scripts that aren't useful at runtime. AFAIK, they are tools that are only useful for building things that link against libPython, which isn't a use case that exists for iOS.

Reply:
pyconfig.h isn't in the config folder, it's in the main include directory.

Reply:
Apologies - the naming threw me off. 

Frameworks include the header files needed to link the library; but they're automatically stripped out of the framework as part of the compilation process, so they're not in the Framework location.

I guess they *could* be included as "data files" and included along with the stdlib... but is there any practical purpose for that? As with my previous answer, my understanding was that this existed entirely to support building things that link against libPython; is there code that relies on reading and parsing a `.h` file at runtime? To me, it seems like extra payload (and thus app size) for no gain.

Reply:
Regardless, the skip comment could definitely be clarified here.